server/server.go

```diff
@@ -47,6 +52,69 @@ func socketConnectionHandler(pool types.Pool, w http.ResponseWriter, r *http.Req
 	go ws.Read()
 }
 
+func uploadImageHandler(w http.ResponseWriter, r *http.Request, db *types.ImageDB) {
+	log.Println("Upload Image endpoint hit!")
+	w.Header().Set("Access-Control-Allow-Origin", "*")
+	switch r.Method {
+	case http.MethodPost:
+		// From https://socketloop.com/tutorials/upload-multiple-files-golang
+		err := r.ParseMultipartForm(200000) // grab the multipart form
```

Comment:
Where did the magic constant `200000` come from? Can we extract it to somewhere and give it a meaningful name at least?

Reply:
Good idea